### PR TITLE
Address tsvart comments

### PR DESCRIPTION
**Pull request by tfpauly**
https://datatracker.ietf.org/doc/review-ietf-intarea-provisioning-domains-09-tsvart-lc-duke-2019-12-21/

### CODE REVIEW

File: draft-ietf-intarea-provisioning-domains.md [hunk 1]
```diff
@@ -778,6 +780,8 @@ It is expected that for some years, networks will have a mixed
 environment of PvD-aware hosts and non-PvD-aware hosts. If there is a
 need to give specific information to PvD-aware hosts only, then it is
 RECOMMENDED to send two RA messages, one for each class of hosts.
+This approach allows for two distinct sets of configuration information
+to be sent in a way that will not disrupt non-PvD-aware hosts.
```
Comment (ekline) [inline draft-ietf-intarea-provisioning-domains.md]: ...and lowers the risk of approaching an MTU limit (from, say, duplicated information)

**Review by ekline — COMMENTED** [review]
LGTM; just a single thought

File: draft-ietf-intarea-provisioning-domains.md (hunk 1 above)
Comment (tfpauly) [reply draft-ietf-intarea-provisioning-domains.md]: Good call, added a sentence!

**Review by ekline — APPROVED** [review]

**Comment by Oryon** [comment]
Good for me.

**Comment by evyncke** [comment]
I like the improved text. Thank you @ekline for your suggested text.

**Review by evyncke — APPROVED** [review]

**Pull request merged**